src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -429,6 +429,7 @@ def __init__(self, version, users, roles):
         self.users = users
         self.version = version
         self.roles = roles
+        self.first_time_login = True
```

Comment:
This could come from a mgr config option.

Reply:
Done. Do I need to document it? @epuertat